### ISSUE TITLE
pixel loss different from paper

### ISSUE THREAD

**Issue by HERIUN** [tzm-tora/Stroke-Based-Scene-Text-Erasing#12]
In the paper
![image](https://github.com/tzm-tora/Stroke-Based-Scene-Text-Erasing/assets/25131767/d38974ce-f8ae-4164-a722-81973b3c05e2)

but in code(src/loss.py)
`hole_loss = self.l1((1-o_mask) * output, (1-o_mask) * gt)`

**Comment by tzm-tora** [comment]
This is the hole loss, and we also add the valid_loss as the entire L_{pixel}, you can recheck it.

**Comment by HERIUN** [comment]
Aha! I appreciate your fast reply.
